### ISSUE TITLE
Error deploying rails4.2 app on TorquBox4 on WildFly

### ISSUE THREAD

**Issue by anichenka** [torquebox/torquebox#210]
Running empty rails4 application on torquebox 4.x.incremental.304 on wildfly-9.0.0.Beta2 (wared together with jruby and gems) and getting

```
"{\"WFLYCTL0080: Failed services\" => {\"jboss.undertow.deployment.default-server.default-host./railsapp\" => \"org.jboss.msc.service.StartException in service jboss.undertow.deployment.default-server.default-host./railsapp: Failed to start service
    Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.io.FileNotFoundException: \\\\Users\\\\USERX\\\\AppData\\\\Local\\\\Temp\\\\wunderboss5916673557745020008\\\\jars\\\\wunderboss-ruby-1.x.incremental.175.jar (The system cannot find the path specified)
```

win7 sp1 64
java 32 build 1.8.0_31-b13
jruby (tried both - 1.7.19 and 9.0.0.0.pre1)
torquebox 4.x.incremental.304
wildfly (tried both - 8.2.0 and 9.0.0.Beta2)

**Comment by bbrowning** [comment]
I tried an empty Rails 4 application on a similar environment of Windows 7, Java 1.8.0_40, JRuby 1.7.19, TorqueBox incremental 304, and WildFly 8.2.0.Final and was not able to reproduce the issue. When I look in the wunderboss* temp directory, I do see a jars\wundrboss-ruby-1.x.incremental.175.jar file.

How are you starting WildFly? I was using the bin\standalone.bat script from the WildFly distribution executed directly in the Command Prompt.

**Comment by anichenka** [comment]
I am starting it the same way - standalone.bat (tried also with full xml config).
I will try to update my java8 to patch 40, to have config similar to yours.

**Comment by anichenka** [comment]
ok, I have updated my jdk to 1.8.0_40, but was getting the same error.

but here is additional info about my environment:

WILDFLY_HOME
d:\dev\wildfly\

JAVA_HONE
c:\Program Files (x86)\Java\jdk1.8.0_40\

So wildfly and java are located on different partitions.

Than I have tried to move wildfly to c:\wildfly\ - and that is it! **Deployed rails app is successfully starting!**

So the error from original post is happening only when java and wildfly are on different partitions.
